ksql-parser/src/main/java/io/confluent/ksql/parser/properties/with/WithClauseProperties.java

```diff
@@ -86,6 +88,32 @@ void validateDateTimeFormat(final String configName) {
     }
   }
 
+  static void validateWindowSizeProperty(final String windowSizeProperty) {
```

Comment:
Feels like this method should be on the `CreateSourceProperties` class, not this shared base class.